config/routes/api/v2.rb

```diff
@@ -291,8 +291,6 @@ class ActionDispatch::Routing::Mapper
           end
         end
 
-        api_resources :upstream_subscriptions, only: :index
```

Comment:
broke a controller test :(

Reply:
Yeah, was waiting to rebase @jlsherrill's PR.